pyproject.toml

```diff
@@ -288,6 +289,7 @@ firebase-admin = "^7.0.0"
 Flask = "^3.0"
 Flask-Babel = "^4.0"
 Flask-Cors = "6.0.1"
+Flask-HTTPAuth = "^4.8.0"
```

Comment:
The codebase already uses Flask's native [`request.authorization`](https://werkzeug.palletsprojects.com/en/stable/datastructures/#werkzeug.datastructures.Authorization) elsewhere, so adding `Flask-HTTPAuth` for this one endpoint is inconsistent and adds a dependency. I'll leave this up to you, but my recommendation would be that we not add this dependency.

Reply:
I see.  I misunderstood you before.  I added a decorator that performs the auth/n for admins using request.authorizaiton.